configure

```diff
@@ -106,7 +106,7 @@ forcesse2=0
 # For CPUs that can benefit from AVX512, it seems GCC generates suboptimal
 # instruction scheduling unless you specify a reasonable -mtune= target
 avx512flag="-mavx512f -mavx512dq -mavx512bw -mavx512vl"
-avx512vnniflag="-mavx512vnni ${avx512flag}"
+avx512vnniflag="${avx512flag} -mavx512vnni -mtune=cascadelake"
```

Comment:
Hardcoding the `-mtune` flag here would break older compilers.

Reply:
Yeah, wikichip seemed to suggest cascade lake was the first cpu supporting VNNI, and first gcc release supporting cascade lake also supported VNNI. But wikichip was misleading.
Will have to fix this for cmake as well.

Reply:
Ice Lake was first Intel CPU to have VNNI instructions. We (currently) use same tuning target for both AVX512 and VNNI.

Reply:
If by currently you mean with this PR then yes. The current develop does not with neither configure nor cmake.

Consider contributing changes to the Microarchitecture support table here: https://en.wikichip.org/wiki/x86/avx512_vnni 😉